buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -276,8 +277,9 @@
     "Label": {
       "value": "$(Build.BuildNumber)"
     },
-    "SignType": {
-      "value": "test"
+    "PB_SignType": {
```

Comment:
PB_ part was not necessary but you've been symmetric so doesn't matter much as long as the PB definition defines a matching variable in the -g section.